Integrations/integration-UrlScan.yml

```diff
@@ -0,0 +1,384 @@
+            }
+            if (resStats.resourceStats[0].countries.length >1) md += " **"+resStats.resourceStats[0].countries[resStats.resourceStats[0].countries.length-1]+")**";
+            md += "\nASN: **"+resStats.ipStats[0].asn.asn+" "+resStats.ipStats[0].asn.description+"**";
+            md += "\nMalicious: **"+resStats.malicious+"**";
+
+            if (resStats.malicious !== 0){
+                dbotScore = 3;
+                ec.File = {};
+                addMalicious(ec, "File", {
+                    MD5: hash,
+                    Malicious: {Vendor: 'URLScanIO', Description: 'Match found in URLScanIO database'}
+                });
+            }else{
+                dbotScore = 1;
+            }
+
+            ec.DBotScore = {Indicator: args.url, Type: 'hash', Vendor: 'URLScanIO', Score: dbotScore};
+            resDis.push ( {'ContentsFormat': formats['json'], 'Type': entryTypes['note'], 'Contents': res, "HumanReadable": md, "EntryContext": ec} );
+            resDis.push(getScreenshot(res.results[0]._id));
+
+
+            return resDis;
+
+        }else{
+            ec.DBotScore = {Indicator: args.url, Type: 'hash', Vendor: 'URLScanIO', Score: dbotScore};
+            md += "\nNo results for this hash"
+            resDis.push( {'ContentsFormat': formats['json'], 'Type': entryTypes['note'], 'Contents': res, "HumanReadable": md, "EntryContext": ec} );
+        }
+
+
+        return resDis;
+    };
+
+
+    var submitUrl = function(scanUrl){
+        if(params.apikey === undefined){
+            return "You need to configure API key in the integration configuration in order to be able to submit a url for a scan.";
+        }
+        url = params.url+"scan/";
+        var body = JSON.stringify({"url": scanUrl, "public": "on"});
+        var res = http(
+            url,
+            {
+                Method: "POST",
+                Headers: {"Content-Type": ['application/json'], "API-Key": [params.apikey]},
+                Body: body
+            },
+            params.proxy
+        );
+
+        if ((res.StatusCode < 200 || res.StatusCode >= 300) || res.success === false) {
+            throw 'Request Failed.\nStatus code: ' + res.StatusCode + '.\nBody: ' + JSON.stringify(res) + '.';
+        }
+
+        var bodyJS = JSON.parse(res.Body);
+
+        var ec = {};
+
+        wait(10);
+        var md = "## Your url has been successfully submitted for scan\n";
+        md += "Scan id: "+bodyJS.uuid;
+        md += "\nUrl to the scan: ["+bodyJS.result+"]("+bodyJS.result+")";
+        ec.UrlScan = {"Url": scanUrl, "Scan":bodyJS.result};
+
+        return ( {'ContentsFormat': formats['json'], 'Type': entryTypes['note'], 'Contents': res, "HumanReadable": md, "EntryContext": ec} );
+    }
+
+
+    switch (command) {
+        // This is the call made when pressing the integration test button.
+        case 'test-module':
+            searchHash("gfdgdfgd");
```

Comment:
looks like something that was left behind by mistake.
change it to a url you know you'll get response (google.com for example)